app/src/components/common/Status.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="status" :class="status.color">
+    {{ status.badge }}
+  </div>
+</template>
+
+<script>
+export default {
+  name: `Status`,
+  props: {
+    status: {
+      type: Object,
```

Comment:
what object is this? can we add a comment?

Reply:
```suggestion
      type: Object, // { color: String, badge: String } // badge is the title of the status
```

badge should be called `caption` or `title` imo

Reply:
yes, weird naming

Reply:
We change it in another PR then

Reply:
OK, just changed it